piker/ui/_display.py

```diff
@@ -109,6 +111,33 @@ def chart_maxmin(
     return last_bars_range, mx, max(mn, 0), mx_vlm_in_view
 
 
+@dataclass
+class DisplayState:
```

Comment:
Love this new `dataclass`

Reply:
Yeah makes it super nice to manual updates - probably will work for remote control decent down the road as well.